iOS/MSQASignIn/src/MSQASignInButton.m

```diff
@@ -58,6 +58,7 @@
 
 static const CGFloat kStandardCornerRadius = 4;
 
+static const CGFloat kFontSizeDeltaForAccessibilityLarger = 4;
```

Comment:
Would it make more sense to use a factor here rather than a delta, so as to increase different font sizes proportionally.

Reply:
Good suggestion! have changed to use a ratio to get the size for larger text, thanks!